src/views/Viewpoint/DetailViewpoint/SidebarTabsContent/sidebar-tabs-content.scss

```diff
@@ -32,18 +32,18 @@
   }
 
   &--related_documents {
-    .bp3-spinner {
-      margin: 0 auto;
-    }
-
-    .bp3-button {
-      background: rgba(216, 216, 216, 0.3);
-      margin-bottom: 0.5em;
+    display: flex;
+    flex-wrap: wrap;
 
-      .bp3-icon,
-      .bp3-button-text {
-        color: #fff;
-      }
+    & a {
+      width: 30%;
+      text-align: center;
+      margin: 2px;
```

Comment:
overridden few lines later

Reply:
I merged two style and I forget to remove duplicates ;-) Done.